README.md

```diff
@@ -106,3 +106,25 @@ The manual process of creating a new version is to:
        ]
     }
     ```
+
+## Overview
+* This repository contains JSON Schema files for all the versions of AsyncAPI specification. You can link to https://json-schema.org/ for reference.
+* These JSON Schema files do not reflect 1:1 the specification and shouldn't be treated as specification itself but rather as a tool for things like validation e.t.c
+* These JSON Schema files should'nt be used as the only tool for validation of AsyncAPI documents because some rules described in the AsyncAPI specification are not described here.
```

Comment:
```suggestion
* These JSON Schema files shouldn't be used as the only tool for validating AsyncAPI documents because some rules described in the AsyncAPI specification are not described with JSON Schema.
```